rbac/internal/urls.py

```diff
@@ -62,6 +62,10 @@
     path("api/v1/openapi.json", openapi, name="openapi"),
 ]
 
+s2s_urlpatterns = [
+    path("_s2s/workspaces/<str:org_id>/ungrouped/", views.retrieve_ungrouped_workspace),
```

Comment:
do we need to state org_id here ? because now we have org id here and also in psk header  - maybe it will be clear for user to have on one place

Reply:
Good catch! Generating the identity requires org_id, so I remove it from this url.